apps/minifront/src/state/staking/index.ts

```diff
@@ -292,7 +362,8 @@ export const createStakingSlice = (): SliceCreator<StakingSlice> => (set, get) =
       // Reload delegation tokens and unstaked tokens to reflect their updated
       // balances.
       void get().staking.loadDelegationsForCurrentAccount();
-      void get().staking.loadUnstakedAndUnbondingTokensByAccount();
+      void get().staking.loadAndReduceBalances();
+      void get().staking.loadUnbondingTokensForCurrentAccount();
```

Comment:
Perhaps all three can just be combined into one? Does it help being separate?

Reply:
see above